stratosphere/ams_mitm/source/fs_mitm/fsmitm_romfsbuild.cpp

```diff
@@ -397,13 +399,11 @@ void RomFSBuildContext::Build(std::vector<RomFSSourceInfo> *out_infos) {
     header->dir_table_ofs = header->dir_hash_table_ofs + header->dir_hash_table_size;
     header->file_hash_table_ofs = header->dir_table_ofs + header->dir_table_size;
     header->file_table_ofs = header->file_hash_table_ofs + header->file_hash_table_size;
-    
-    const size_t metadata_size = this->dir_hash_table_size + this->dir_table_size + this->file_hash_table_size + this->file_table_size;
```

Comment:
Why remove this?

Reply:
Oh, you moved it up. Never mind.

Reply:
I didn't, just moved it up there since the size is needed for malloc anyway